core/evancore.vue.libs/src/component.ts

```diff
@@ -20,19 +20,47 @@
 // vue imports
 import Component from 'vue-class-component';
 import Vue from 'vue';
-import VueRouter, { Route } from 'vue-router';
+import VueRouter, { Route, RawLocation } from 'vue-router';
 
 // evan.network imports
 import * as dappBrowser from '@evan.network/ui-dapp-browser';
+import { Runtime } from '@evan.network/api-blockchain-core';
 import { getDomainName } from './utils';
 import { StartedDAppInfo } from './interfaces';
 
+declare module 'vue/types/vue' {
```

Comment:
Can we move this declaration to our general vue types file?

https://github.com/evannetwork/ui-dapps/blob/feature/CORE-849-digital-twin-general-overview/scripts/dapp/vue-shim.d.ts